### PR TITLE
Spec n days

### PR DESCRIPTION
**Pull request by adsantos**

### CODE REVIEW

File: input_n_days.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+    "items": [
+        {
+            "id": 1,
+            "content": "Call David.",
+            "dueDate": "2015-07-11T18:25:43.511Z",
+            "priority": null,
```
Comment (Garee) [inline input_n_days.json]: I think we should default every item to a `priority` of `4` (i.e. 'no priority') rather than `null`.

**Comment by adsantos** [comment]
Ready for review

File: spec_n_days.json [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+                {
+                    "id": "1",
+                    "freezeTime": "2015-07-10T18:25:43.511Z",
+                    "currentUser": null,
+                    "context": "When asked for tasks in the next 3 days in input_overdue_no_due_date.json",
+                    "query": "3 days",
+                    "testFile": "input_overdue_no_due_date.json",
+                    "assertSections": [
+                        {
+                            "name": "10th of July",
```
Comment (Garee) [inline spec_n_days.json]: I think we should use the current naming scheme for `n days` where the sections are named `today`, `tomorrow`, `wednesday`, `thursday` etc.

File: spec_n_days.json [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+                            "type": "Filtered",
```
Comment (Garee) [inline spec_n_days.json]: The type will be `Daily` which allows drag/drop functionality.

File: spec_n_days.json [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+                            "priority": null,
+                            "date": null,
+                            "project": null,
+                            "items": [2,3],
+                            "isReorderable": false,
+                            "labels": null
+                        }
+                    ]
+                }
```
Comment (Garee) [inline spec_n_days.json]: Should we treat all `n < 1` as an invalid query? Probably need some tests for those too.

File: spec_n_days.json (hunk 2 above)
Comment (adsantos) [inline spec_n_days.json]: In the web, if I type 30 days, I only get one section and it's still the day of the week. In front of the week day, it has the date, but that's another field. In conclusion, I'll change the section name to always have the day of the week.

**Comment by adsantos** [comment]
All done!

File: spec_n_days.json [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+                {
+                    "id": "7",
+                    "freezeTime": "2015-07-10T18:25:43.511Z",
+                    "currentUser": null,
+                    "context": "When asked for tasks in the next 0 days in input_overdue_no_due_date.json",
+                    "query": "0 days",
+                    "testFile": "input_overdue_no_due_date.json",
+                    "assertSections": []
+                },
+                ,
```
Comment (Garee) [inline spec_n_days.json]: Typo here - extra comma!

Comment (adsantos) [inline spec_n_days.json]: Done

**Pull request merged**